sirepo/sim_data/__init__.py

```diff
@@ -677,11 +727,25 @@ def split_jid(jid):
 def _init():
     global cfg
 
+    sirepo.job.init()
     cfg = pkconfig.init(
         lib_file_resource_only=(False, bool, 'used by utility programs'),
         lib_file_list=(None, lambda v: pkio.read_text(v).split('\n'), 'directory listing of remote lib'),
         lib_file_uri=(None, str, 'where to get files from when remote'),
+        supervisor_db_file_uri=(None, str, 'where to get/put db files from/to supervisor'),
+        supervisor_db_file_key=(None, str, 'key for supervisor file access'),
+    )
+
+
+def _request(method, uri, data=None):
+    return requests.request(
+        method,
+        uri,
+        data=data,
+        verify=sirepo.job.cfg.verify_tls,
+        headers=PKDict(Authorization=f'Bearer {cfg.supervisor_db_file_key}')
```

Comment:
should be wrapped.

Reply:
https://github.com/radiasoft/sirepo/pull/3266/commits/2b4b6559d8022c36e74cdadc103a85f965e9927d